crypto-ffi/bindings/js/tsconfig.json

```diff
@@ -1,29 +1,41 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "nodenext",
-    "allowJs": true,
+ // Taken from https://bun.sh/docs/typescript#suggested-compileroptions
+ 
+ // Environment setup & latest features
```

Comment:
Nit: these comments are unaligned, while the rest are aligned, which looks weird.